network/network.go

```diff
@@ -230,6 +230,8 @@ func (n *Network) Start() error {
 		n.connectionManager.Connect(bootstrapNode)
 	}
 
+	log.Logger().Infof("started %s", ModuleName)
```

Comment:
intentional, of leftover debugging?

If we want this logged for all engines, we'd better do it in `System`